Frameworks/CoreMotion/CMMotionManager.mm

```diff
@@ -190,18 +180,18 @@ - (void)startAccelerometerUpdatesToQueue:(NSOperationQueue*)queue withHandler:(C
         self.accelerometerQueue = queue;
 
         self.accelerometerToken =
-            [self.accelerometer addReadingChangedEvent:^void(WDSAccelerometer* sender, WDSAccelerometerReadingChangedEventArgs* e) {
```

Comment:
Will the C++ lambda here extend the lifetime of queue if it gets released before the reading callback fires? I know blocks will explicitly do so.

This could be a concern for all async objective-c things, and necessitate the use of AutoId/StrongId with its copy constructor to ensure object lifetimes.